Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +11,11 @@ static void Main(string[] args)
             // - FindPersonWithBiggestLoss
             // - FindRichestPerson
             // - FindMostPoorPerson
+            Console.WriteLine(TextTable.Build(BalanceStats.FindHighestBalanceEver(PeoplesBalances.Balances),3));
+            Console.WriteLine(TextTable.Build(BalanceStats.FindPersonWithBiggestLoss(PeoplesBalances.Balances),3));
+            Console.WriteLine(TextTable.Build(BalanceStats.FindRichestPerson(PeoplesBalances.Balances),3));
+            Console.WriteLine(TextTable.Build(BalanceStats.FindMostPoorPerson(PeoplesBalances.Balances),3));
```

Comment:
I would put 0 blank lines in beween those text tables- because they are the same/similar thing.
However, I would put a blank line after Console.ReadKey() as a way to separate it from the rest, indicating that it's a completely different thing/intention

Reply:
You're, code should be clean and more readable.